daffodil-lib/src/main/resources/org/apache/daffodil/xsd/XMLSchema_for_DFDL.xsd

```diff
@@ -1757,11 +1738,19 @@ this to validate when we load a DFDL Schema.
     </xsd:annotation>
   </xsd:element>
 
-  <xsd:element name="enumeration" id="enumeration" type="sub:noFixedFacet">
+  <xsd:element name="enumeration" id="enumeration">
     <xsd:annotation>
       <xsd:documentation
         source="http://www.w3.org/TR/xmlschema-2/#element-enumeration" />
     </xsd:annotation>
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="sub:facet">
+          <xsd:attributeGroup ref="dfdl:RepValuesAGQualified" />
+          <xsd:attributeGroup ref="sub:anyOther" />
```

Comment:
This sub namespace.... do we still use it? At one time we had this hack going about not using xs or xsd for the subset-of-XML-Schema schema, but I gave up on that long ago. Is sub still essential?

Reply:
I'm not really sure. It's only used in XMLSchema_for_DFDL.xsd, and the sub prefix has the same namespace as the xsd and xs prefixes, so  I don't think it really provides any benefit. That said, it shows up 261 times in that file, so it'd be a pretty big diff and would hide the changes in this file. I'm fine with removing it, but we should do that as a separate bug if we want.